notebook_sandbox/wavecal_1d_shane.ipynb

```diff
@@ -0,0 +1,283 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "051529f9-7caf-428d-969c-51aa6c2f8f83",
+   "metadata": {},
+   "source": [
+    "# Specreduce 1D wavelength calibration example 1\n",
+    "\n",
+    "**Author:** Hannu Parviainen </br>\n",
+    "**Modified:** 16 Jan. 2025"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "463038ad-ae02-4484-8226-36df99fbf810",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "7853ed1c-5b05-42a2-9413-4054769c6032",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import astropy.units as u\n",
+    "import numpy as np\n",
+    "\n",
+    "from astropy.io.fits import getdata\n",
+    "from astropy.nddata import StdDevUncertainty\n",
+    "from matplotlib.pyplot import setp, subplots, close, rc\n",
+    "from specutils import Spectrum1D\n",
+    "\n",
+    "from specreduce.lswavecal1d import WavelengthSolution1D\n",
+    "\n",
+    "rc('figure', figsize=(11,3))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "e3dd2a51-ef1a-4501-9bfb-62166676e88f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "flux = getdata('shane_kast_blue_600_4310_d55.fits', 1).astype('d')\n",
```

Comment:
need to provide the path `../docs/wavelength_calibration/wavelength_calibration/`, and also it looks like the file in that directory is different shane_kast_blue_600_4310_d55.fits so either change the filename to that one or add the correct file.